styles.css

```diff
@@ -103,3 +103,423 @@ ul {
   list-style-type: none;
   display: flex;
 }
+
```

Comment:
- [ ] Please add the CSS property `display:grid` to the main container (`<div class="grid-container">`) as required in the project requirement
![Screenshot from 2023-03-24 16-24-46](https://user-images.githubusercontent.com/91142432/227889880-ad19f125-e81b-4af4-948c-5c241d67853a.png)